core/commonMain/src/ArgParser.kt

```diff
@@ -195,17 +232,32 @@ open class ArgParser(val programName: String, var useDefaultHelpShortName: Boole
     }
 
     /**
-     * Add argument with single nullable value and get delegator to its value.
+     * Declares an argument and returns an object which can be used to access the argument value
+     * after all arguments are parsed or to delegate a property for accessing the argument value to.
+     *
+     * By default, the argument supports only a single value, is required, and has no default value,
+     * therefore its value's type is `T`.
+     *
+     * You can alter the argument properties by chaining extensions for the argument type on the returned object:
+     *   - [AbstractSingleArgument.default] to provide a default value that is used when the argument is not specified;
+     *   - [SingleArgument.optional] to allow omitting the argument;
+     *   - [AbstractSingleArgument.multiple] to allow specifying the argument several times, but no more than the given number;
+     *   - [AbstractSingleArgument.vararg] to allow specifying the last argument an unlimited number of times.
```

Comment:
May be not "the last argument", but " values for last argument"?

Reply:
Rephrased: "to allow specifying an unlimited number of values for the _last_ argument."